app/config.py

```diff
@@ -54,6 +54,20 @@ def all_queues():
             QueueNames.REPORT_REQUESTS_NOTIFICATIONS,
         ]
 
+    def non_worker_queues():
```

Comment:
I think a better name for this would be either `non_api_queues` or `external_queues`, because we still call the ecs tasks that consume them "workers".

Reply:
I did call it non_api_queues to begin with... but then I saw there is code that publishes to those queues.

Reply:
Renamed to external_queues